src/drive/targets/manifest.webapp

```diff
@@ -101,7 +101,7 @@
     "dacc": {
       "type": "node",
       "file": "services/dacc/drive.js",
-      "trigger": "@every 720h"
+      "trigger": "@monthly on the 1-3 before 5am"
```

Comment:
1-3 is not clear for me

Let's try, something like this?

```suggestion
      "trigger": "@monthly on the 1st to 3rd of the month before 5am"
```

Reply:
I don't know if it's clearer, but it definitely won't work ;) https://github.com/cozy/cozy-stack/blob/master/docs/jobs.md#monthly-syntax

Reply:
oops, big confusion. No worry. I resolve